### DISCUSSION TITLE
Installing argdown-cli

### DISCUSSION THREAD

**Discussion by mikegeiger** [christianvoigt/argdown#257] [Q&A]
Hi Christian,

a friend of mine came across your tool and we would like to play a little with argdown for some debates. We both struggle installing it, though. I was wondering if you could support us to get it up and running?
Doing so, we could even work together and putting an install script together. This might be useful for more people.
By the way, we tried LinuxMint and MacOS.

Kind Regards
Mike

**Comment by christianvoigt** [comment]
Hi Mike, I recommend to [install VS Code and the Argdown extension](https://argdown.org/guide/installing-the-vscode-extension.html#activating-the-argdown-theme-in-vs-code). It is much easier to install and use. The commandline tool is for users who prefer other editors or for more advanced use cases. 

However, it would be good to know what problems exactly you were running into: Have you followed the [installation guide](https://argdown.org/guide/installing-the-commandline-tool.html)? 

I think there is a small error in there. If you want to install argdown-cli globally with npm you have to use the "-g" flag: 

```bash
npm install -g @argdown/cli
````

I do not really see the need for an install script as this is already only one line (once you have installed Node.js).

**Comment by mikegeiger** [comment]
Oh wow, thank you for the fast answer. That actually solved it! I guess npm installed it in a local directory before and couldn't resolve a bunch of dependencies.

Regarding the VS Code extension, yes I intend to use it, too. I wanted to experiment with the cli commands nonetheless.
Thank you for your support!
Mike

**Comment by christianvoigt** [comment]
Cool, I will open another issue about fixing the installation instructions.
